### ISSUE TITLE
How to process my own video into training dataset or test dataset?

### ISSUE THREAD

**Issue by Aruen24** [qiuyu96/CoDeF#30]

**Comment by ken-ouyang** [comment]
You can use FFMPEG or other tools to extract your videos to video frames and follow the data structures in README. We will soon release the preprocess code for other steps like masks or optical flows. 

I think there is no need to divide the training and test dataset as our model is optimized to overfit to the whole video.

**Comment by lininghuhu** [comment]
> You can use FFMPEG or other tools to extract your videos to video frames and follow the data structures in README. We will soon release the preprocess code for other steps like masks or optical flows.
> 
> I think there is no need to divide the training and test dataset as our model is optimized to overfit to the whole video.

looking foward to the code coming soon

**Comment by Aruen24** [comment]
@ken-ouyang OK，Thanks  waiting the preprocess code for other steps like masks or optical flows

**Comment by qiuyu96** [comment]
Hi, we have uploaded data preprocessing scripts. See details [here](https://github.com/qiuyu96/CoDeF#customize-your-own-data) .
